src/modules/login/index.tsx

```diff
@@ -0,0 +1,155 @@
+import {Image, ImageBackground, Keyboard, KeyboardAvoidingView, Platform, StyleSheet, View} from "react-native";
+import React, {useReducer, useState} from "react";
+import {handleReducerChange, stateReducer} from "../../utils/ReactReducer";
+// @ts-ignore
+import logo from '../../../assets/logo-img.png'
+// @ts-ignore
+import imageBackground from '../../../assets/login-background.jpg'
+import {layout} from "../../styles/layout";
+import TextInput from "../../components/TextInput";
+import SocialLine from "./components/socialLine";
```

Comment:
you have index file for it, and can handle import in one line` {c1, c2, c3} from '../../components'`